README.md

```diff
@@ -27,7 +27,8 @@ This repository contains the sourcecode for the [MAAS](https://maas.io) web app,
 
 Community contributions are most welcome, and there are a number of ways to participate:
 
-- [Submit bugs and feature requests](https://github.com/canonical/maas-ui/issues)
+- [Submit bugs and feature requests](https://bugs.launchpad.net/maas)
```

Comment:
feature requests should be directed to Discourse

Reply:
maybe refer to https://maas.io/docs/how-to-review-and-report-bugs instead ?

Reply:
done